library/examples/hate_speech.py

```diff
@@ -77,7 +77,7 @@
         num_train_epochs=2,
         weight_decay=0.01,
         save_strategy="epoch",
-        push_to_hub=True,
+        push_to_hub=False,
```

Comment:
If this is false you cannot upload the model to ``hf``! Why did you set this to ``False``?